model/ExpandedLicensing/Classes/NoneLicense.md

```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NoneLicense
+
+## Summary
+
+A concrete subclass of AnyLicenseInfo used by Individuals in the ExpandedLicensing profile.
```

Comment:
Should this reference `IndividualLicense` instead of `AnyLicenseInfo`? And also specify which Individual uses it? E.g.:

"A concrete subclass of IndividualLicense used by the NoneLicense Individual in the ExpandedLicensing profile."

Reply:
We'll solve this by getting rid of `IndividualLicense`